core-kotlin-modules/core-kotlin-5/src/test/kotlin/com/baeldung/openconstructor/LanguageTest.kt

```diff
@@ -0,0 +1,27 @@
+package com.baeldung.openconstructor
+
+import org.junit.Ignore
+import org.junit.Test
+import org.junit.jupiter.api.Assertions.*
+
+class LanguageTest {
+
+  @Test
+  @Ignore
+  fun `Non-Lazy Languages should return their alphabet sizes`() {
+    val languageA: NonLazyLanguage = NonLazyEnglish()
+    val languageB: NonLazyLanguage = NonLazySpanish()
+
+    assertEquals(languageA.alphabetSize, 26)
+    assertEquals(languageB.alphabetSize, 27)
```

Comment:
Note: these assertions are the wrong way around. It's `assertEquals(expected, actual)` where you're doing `assertEquals(actual, expected)` here

Reply:
done